deep-learning/_posts/2018-03-22-detectionDistillation.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: review
+title:  "Knowledg Distillation on Object Detection"
+tags:   "Deep Learning"
+author: Faezeh Amjadi
+pdf:    https://papers.nips.cc/paper/6676-learning-efficient-object-detection-models-with-knowledge-distillation.pdf
+cite:
+  authors: "Guobin Chen, Wongun Choi, Xiang Yu, Tony X. Han, Manmohan Krishna Chandraker"
+  title:   "Learning Efficient Object Detection Models with
+Knowledge Distillation"
+  venue:   "NIPS2017"
+---
+
+### Description
+	
+They proposed an end-to-end trainable framework for learning multi-class object detection through knowledge distillation. A teacher network (a pretrained network with more parameters and deeper) supervises and leads to better student performance which the knowledge is received from teacher is more informative. 
+
+<img src="/deep-learning/images/detecting_KD/0.png" width="800">
+
+
+Faster-RCNN is their object detection framework which is composed of two modules:1) a region proposal network(RPN) that generates object proposals, and 2) RPN inputs to a classification and regression network(RCN) that return the detection score and bounding boxes.They learn a student object detector by using Knowledge of a high capacity teacher detection network.There is three steps:
+
+1) Encourage the feature representation of a student network is similar to that of the teacher network.
+2) The soft teacher targets in classification and regrestion is applied to transfer knowledge to student.
+3) The student have a more loss by groundtruth and its prediction.   
```

Comment:
The student network has loss computed between its prediction and the groundtruth.